jobs/cam/plots/JEVS_CAM_PLOTS

```diff
@@ -65,6 +65,14 @@ else
     export VDATE=$PDYm1
 fi
 
+if [ $MODELNAME = nam_firewxnest ]
```

Comment:
Should this `$MODELNAME = nam_firewxnest` if block be included up in the `${COMPONENT} == "cam"` if block, as an `elif` between `$MODELNAME = href` and `else`?

Reply:
I feel that there will need to be an extensive cleaning up of this block in the final version, so I left it out.  I think this is a later task.  For now, this will work.